src/shogun/features/StringFeatures.cpp

```diff
@@ -1973,8 +1973,8 @@ template<> void CStringFeatures<sg_type>::load(CFile* loader)		\
 																			\
 	SG_SET_LOCALE_C;													\
 	SGString<sg_type>* strs;												\
-	int32_t num_str;														\
-	int32_t max_len;														\
+	index_t num_str;														\
+	index_t max_len;														\
 	loader->f_load(strs, num_str, max_len);									\
```

Comment:
Hi @vigsterkr , I got a bunch of warnings in this LOAD macro like https://pastebin.mozilla.org/9029036. All of them are about` 'num_str' is uninitialized when used here`. Do I need to give it a value like '0' or ?